lib/src/builder.dart

```diff
@@ -135,6 +136,20 @@ class AggregateTestBuilder extends Builder {
     // Don't generate an empty aggregate test.
     if (imports.isEmpty) return;
 
+    if (_config.randomizeAggregation) {
+      log.fine('randomizing test order');
+      final randomizedMains = mains.toString().split(';\n')
+        ..removeWhere((element) => element.isEmpty);
+      var seed = _config.testShuffleSeed;
+      log.warning(
+          'Shuffling test order with --test-randomize-ordering-seed=$seed');
```

Comment:
I think this would be better as an `info` level log

Reply:
I left this as a warning because otherwise I couldn't seem to get it to display in the output of the test run

Reply:
Yeah, the default behavior of `pub run build_runner` is to log `info` level logs such that only the most recent one is visible. In other words, they log it, and then overwrite it as soon as there is another log. But if you run with `-v|--verbose`, then each info log will be on its own line. I think that's probably what we want here because in most cases people won't care about this log but can see it if they need to, and a warning might make them think something is wrong/unexpected when it isn't.

Reply:
Is there  a way to specify that per builder? Asking because that will end up being a lot of logging

Reply:
There is not, but you could always `| grep ` by the logger name